app/src/main/java/org/ole/planet/myplanet/services/SharedPrefManager.kt

```
@@ -143,11 +143,37 @@ class SharedPrefManager @Inject constructor(
+        if (value.isNotEmpty() && value != default) {
+            setSecurePref(key, value)
+        }
+        return value
+    }
+
+    private fun setSecurePref(key: String, value: String) {
+        pref.edit {
+            try {
+                val enc = org.ole.planet.myplanet.utils.SecurePrefs.encryptString(context, value)
+                putString(key, "enc:$enc")
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)